src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -109,7 +110,30 @@ public IPrincipal User
         }
 
         [Activate]
-        public ViewDataDictionary ViewData { get; set; }
+        public ViewDataDictionary ViewData
```

Comment:
Add a doc comment explaining the values of ViewData with remark for what happens in the unit test scenario